extensions/azure/cosmos/contract-definition-store-cosmos/src/test/java/org/eclipse/dataspaceconnector/contract/definition/store/CosmosContractDefinitionStoreIntegrationTest.java

```diff
@@ -47,6 +48,8 @@
     private TypeManager typeManager;
     private CosmosContractDefinitionStore store;
 
+    private static final String PARTITION_KEY_NEW = "test-ap-id1-new";
```

Comment:
nit: may be name of the variable could be improved. Understand it is because of the existing values.